libs/TaskManager.js

```diff
@@ -149,6 +153,19 @@ module.exports = class TaskManager{
 			if (task){
 				this.addToRunningQueue(task);
 				task.start(() => {
+					if(task.webhook && task.webhook.length > 3){
```

Comment:
It times out not really affecting the completion of the task but prolly better to validate valid when adding the URL and then maybe log the error is all